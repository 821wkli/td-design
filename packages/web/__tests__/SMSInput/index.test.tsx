import React from 'react';
import { shallow ,mount} from 'enzyme';
import CheckCode from '../../components/sms-input';

describe('测试CheckCode组件', () => {
  it('1. 成功渲染', () => {
    const wrapper = shallow(<CheckCode type={1}/>);
    expect(wrapper).toMatchSnapshot();
  });

  it('2. 测试一开始的展示文字', () => {
    const wrapper = mount(<CheckCode type={1} phone='123456789'/>);
    expect( wrapper.find('a').text()).toBe('获取验证码')
  });
  
})