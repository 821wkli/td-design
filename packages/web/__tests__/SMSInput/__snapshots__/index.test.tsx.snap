// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试CheckCode组件 1. 成功渲染 1`] = `
<Input
  addonAfter={
    <a
      onClick={[Function]}
    >
      获取验证码
    </a>
  }
  className="code"
  placeholder="请输入短信校验码"
  type="text"
/>
`;
